DemoApp/android/app/build.gradle

```diff
@@ -84,7 +84,6 @@ def enableProguardInReleaseBuilds = false
 
 android {
     compileSdkVersion 23
-    buildToolsVersion "23.0.1"
```

Comment:
I don't understand why remove `buildToolsVersion`, is it no longer needed other changes in this PR?

Reply:
https://developer.android.com/studio/releases/build-tools

> If you're using Android plugin for Gradle 3.0.0 or higher, your project automatically uses a default version of the build tools that the plugin specifies.

Reply:
thanks @MatkovIvan, make sense